articles/libraries/lock/v10/index.md

```diff
@@ -58,7 +57,7 @@ If you are using browserify or webpack to build your project and bundle its depe
 
 ## Usage
 
-### Implementing Lock
+### 1) Implementing Lock
```

Comment:
Also, `implementing` is very generic. You are actually initializing in this snippet, right? Maybe update the header and add a sentence or two to clarify things.